data-prepper-plugins/prometheus-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/prometheus/configuration/PrometheusSinkConfiguration.java

```diff
@@ -108,6 +110,14 @@ public class PrometheusSinkConfiguration {
     @JsonProperty("http_retry_interval")
     private Duration httpRetryInterval = DEFAULT_HTTP_RETRY_INTERVAL;
 
+    @JsonProperty("encoding")
+    private String encoding = DEFAULT_ENCODING;
+
+    @JsonProperty("content-type")
```

Comment:
This project uses underscores for YAML configuration property names. Please change to `content_type`.

Reply:
@dlvenable Resolved